test_redirects/map_globalconf.py

```diff
@@ -213,8 +213,8 @@
     url_test('/firefox/mobile/faq/?os=firefox-os', '/firefox/os/faq/'),
 
     # Bug 986174
-    url_test('/firefox/mobile/features/', '/firefox/android/'),
-    url_test('/firefox/mobile/faq/', '/firefox/android/faq/'),
+    url_test('/{m,{firefox/,}mobile}/features/', '/firefox/android/'),
+    url_test('/{m,{firefox/,}mobile}/faq/', '/firefox/android/faq/'),
```

Comment:
will you explain the nested grouping in the first arg on this line and the line above please? it seems like it would test the following:
`/m/faq/`
`/firefox//faq/`  <-- note the double slash
`/mobile/faq`